### ISSUE TITLE
pipelines: yarn integ fails to deploy 

### ISSUE THREAD

**Issue by dnz-bdeboer** [aws/aws-cdk#15915]
I'm trying to run `yarn integ` on the pipelines directory. This fails with:

```
Failed resources:
 2:10:33 pm | CREATE_FAILED        | AWS::CodePipeline::Pipeline | Pipeline/Pipeline (Pipeline9850B417) Internal Failure
```

There is no detail on the internal error, and I have been unable to see what could be the cause here. But I expected the code to run out of the box.

### Reproduction Steps

Just run `yarn integ integ.newpipeline.js` in the `packages/@aws-cdk/pipelines` directory.

### What did you expect to happen?

Expected to be able to deploy to test changes I'm making.

### Environment

  - CDK cli: 1.116
  - Basically on latest master.
  - node.js: v16.0.0
  - OS: Mac
  - Typescript: not sure, probably what is in `package.json`? Have installed 4.0.3 globally.

--- 

This is :bug: Bug Report

**Comment by rix0rrr** [comment]
https://docs.aws.amazon.com/cdk/api/latest/docs/pipelines-readme.html#pipeline-internal-failure

**Comment by dnz-bdeboer** [comment]
Thanks @rix0rrr , must have missed this new section in the README! What about adding a brief comment to `integ.newpipeline.ts` just above the `input` action about such a token?

But even better to make this work out of the box: what about using an S3 bucket as provider, and the S3 bucket is simply world-readable. So no one needs to bother with access tokens in order to run this test.

Because even with your guidance I have not yet managed to create an access token that works. Have done two attempts, both failed. I'll probably figure it out, but it's not the great first time experience that is possible.
